src/server/pkg/storage/chunk/storage.go

```diff
@@ -0,0 +1,65 @@
+package chunk
```

Comment:
Small thing on organization, it seems like if we're going to have this under `pkg/storage` we should probably relocated `pkg/obj` to `pkg/obj/storage` as well. That or both should be their own top level directories.

Reply:
I don't think this will ultimately rest under `pkg`. I was just putting off that decision until later. I feel like where this should be will make more sense over time.